### ISSUE TITLE
Encoder Doesn't Start

### ISSUE THREAD

**Issue by TahareeVR** [sorayuki/obs-multi-rtmp#402]
I just downloaded this plug-in, as I wanted to start Multicasting to Twitch and Kick, however, no matter what I do the plug-in won't start. It keeps giving me the error "Failed to start output. It's possible encoder doesn't start". The issue with that error is my encoder does start, it works perfectly fine. So I don't know what's wrong. I have tried changing which encoders it runs off of, I've tried testing this in Broadband testing and live, and I even tried going in and changing the settings, thinking it was either a frame rate or bit rate issue, but still no dice. Any idea as to why it's doing this and how to fix it?

![Capture](https://github.com/sorayuki/obs-multi-rtmp/assets/158778446/2663056f-a018-4ca4-a648-e5a99799b057)

**Comment by pixeldune** [comment]
Same here. Send help! Both plug in and OBS is latest

**Comment by TahareeVR** [comment]
> Same here. Send help! Both plug in and OBS is latest

If you haven't figured out how to get it fixed I was able to get it to work by going to the second to last update of this plug in and it works perfectly!

**Comment by TahareeVR** [comment]
> Same here. Send help! Both plug in and OBS is latest

Download the second to last update and you'll be able to get it to run. Sorry, didn't mean to close it.

**Comment by v4u6h4n** [comment]
Same problem here. Anyone know what the cause is?

**Comment by TahareeVR** [comment]
@v4u6h4n I am not sure, I think the newest update had a bug in it. I am using  the obs-multi-rtmp 0.5.0.3 for OBS 30.0.0 version and it works perfectly. If you go to the releases section and scroll down, then you can download that version and itll work.

**Comment by v4u6h4n** [comment]
@TahareeVR 

Yeah I don't think versions above 30.0.0 are supported yet. Damn. I need the new pipewire support so don't think I can revert to 30.0.0. But thanks for the heads up :-)

**Comment by reliznous** [comment]
Im getting this same issue on obs 31.1.2 using current using 0.7.2, using youtube on h.265 and trying to start twitch on h.264 and it will not start....
